packages/theme-chalk/src/date-picker/date-picker.scss

```diff
@@ -64,6 +64,8 @@
     line-height: 22px;
     text-align: center;
     cursor: pointer;
+    background: none;
```

Comment:
Why is css change necessary

Reply:
Original span converted to button - which is why need to remove background